operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/kubernetes/KubernetesDependentResource.java

```diff
@@ -37,10 +37,18 @@ public abstract class KubernetesDependentResource<R extends HasMetadata, P exten
     implements ConfiguredDependentResource<KubernetesDependentResourceConfig<R>> {
 
   private static final Logger log = LoggerFactory.getLogger(KubernetesDependentResource.class);
+
+  @SuppressWarnings("rawtypes")
+  private static final SSABasedGenericKubernetesResourceMatcher defaultMatcher =
```

Comment:
maybe this could help with the warning:
```suggestion
  private static final SSABasedGenericKubernetesResourceMatcher<?> defaultMatcher =
```

Reply:
so it might be also `R` since it's set only on line 50.

Reply:
This won't work on static fields.